defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-java_version: "7u51-2.4.6-1ubuntu4"
+java_version: "7u79-2.5.6-0ubuntu1.14.04.1"
```

Comment:
We should default this to a version glob of `7u79*`, similar to the NTP role in https://github.com/azavea/ansible-ntp/commit/e22d59a7c405fedbf8a5c8244058b84b0fba9e76.